roles/ipfs/tasks/main.yml

```diff
@@ -1,28 +1,34 @@
-- name: create download folder for go-ipfs
+---
+- name: create download folder for ipfs-update
   become: yes
   file:
     state: directory
     owner: root
     group: root
-    dest: /opt/go-ipfs/{{ipfs_version}}
+    dest: /opt/ipfs-update/{{ipfs_update_version}}
 
-- name: download and unpack go-ipfs
+- name: download and unpack ipfs-update
```

Comment:
wondering, is there something that makes it specially more desirable to use ipfs-update?

Reply:
`ipfs-update` [applies data migrations when upgrading](https://ipfs.io/docs/install/#upgrading-using-ipfs-update), although I suppose that it's taken care of by running ipfs with the `--migrate` flag on each run?

Reply:
@jeffbr13 I *think* it's taken care of by --migrate doing pretty much the same, not %100 sure though. I'll find out.

There is a small advantage to the previous approach which is using a custom dist url (which might potentially have a custom ipfs version in it). While `ipfs-update` attempts to use `localhost:5001`, i don't think you can point it to a different gateway, and since you are installing ipfs you are unlikely running it locally. But as I say, it's small. I do use it more for cluster.

Reply:
Ah I see. I should probably have split this up into more PRs to discuss separately, but I didn't expect this much of a response! I'll revert back to the previous method as you point out the benefit.